recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: pip install --no-deps .
```

Comment:
I meant the bogus missing `six` from https://github.com/conda-forge/nbdime-feedstock/pull/2#issuecomment-271514653

Hopefully we won't need to specify the entrypoints with the next version of `conda-build` (no pressure @msarahan :stuck_out_tongue_winking_eye:)